MoreLinq/RandomSubset.cs

```diff
@@ -23,6 +23,45 @@ namespace MoreLinq
 
     public static partial class MoreEnumerable
     {
+        /// <summary>
+        /// Returns a sequence of random elements from the original sequence.
+        /// </summary>
+        /// <typeparam name="T">The type of source sequence elements.</typeparam>
+        /// <param name="sequence">
+        /// The sequence from which to return random elements.</param>
+        /// <returns>
+        /// A random sequence of elements in random order from the original
+        /// sequence.
+        /// </returns>
+        public static IEnumerable<T> Shuffle<T>(this IEnumerable<T> sequence)
+        {
+            return Shuffle(sequence, new Random());
+        }
+
+        /// <summary>
+        /// Returns a sequence of random elements from the original sequence.
```

Comment:
This summary is a duplicate of the other. It can be rendered unique by saying that, “An additional parameter specifies a randomizer”.